app/controllers/application_controller.rb

```diff
@@ -3,7 +3,18 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  def  aaa
-      1 + 1
+  def  aaa (a  ,  b)
```

Comment:
Unused method argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used. You can also write as `aaa(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used. You can also write as `aaa(*)` if you want the method to accept any arguments but don't care about them.<br>Unnecessary spacing detected.<br>Do not put a space between a method name and the opening parenthesis.<br>Space found before comma.